config/initializers/zz_codidact_sites.rb

```diff
@@ -4,7 +4,13 @@
   if Rails.env.development? || Rails.env.test?
     []
   else
-    response = Net::HTTP.get_response(URI('https://codidact.com/communities.json'))
+    uri = URI('https://codidact.com/communities.json')
+    req = Net::HTTP::Get.new(uri)
```

Comment:
Let's maybe take the opportunity and untie development from codidact being up by making this configurable and not hardcoded? Also makes sense if we ever switch domains or restructure. If I am able to get some time off for that, I can push here or barring that - separately.